src/writecabrillo.c

```diff
@@ -695,7 +695,7 @@ void write_adif_header(FILE *fp) {
     fprintf(fp, "Contest Name: %s\n", whichcontest);
     fputs("<adif_ver:4>3.10\n", fp);
     fputs("<programid:3>TLF\n", fp);
-    fprintf(fp, "<programversion:%ld>%s\n", strlen(VERSION), VERSION);
+    fprintf(fp, "<programversion:%ld>%s\n", (long int)strlen(VERSION), VERSION);
```

Comment:
No point using a long int for abt 10. A simple `programversion:%d` would suffice.

Reply:
That was my thought.  But, someone (Tom?) tossed that `l` in there so I worked around that.  :)